solr/core/src/java/org/apache/solr/search/CollapsingQParserPlugin.java

```diff
@@ -218,7 +245,7 @@ public static GroupHeadSelector build(final SolrParams localParams) {
     public String hint;
     private boolean needsScores = true;
     private boolean needsScores4Collapsing = false;
-    private int nullPolicy;
+    private NullPolicy nullPolicy;
     private Set<BytesRef> boosted; // ordered by "priority"
     public static final int NULL_POLICY_IGNORE = 0;
```

Comment:
These constants can be removed and everything routed through the NullPolicy enum. Same for the `NULL_COLLAPSE` string above and others.

Reply:
@madrob Missed your comment somehow. Added the change that you have suggested. Can you please review this now?